src/main/kotlin/no/nb/mlt/wls/order/controller/OrderController.kt

```diff
@@ -33,7 +33,8 @@ class OrderController(val orderService: OrderService) {
             responseCode = "400",
             description =
                 """Order payload is invalid and was not created.
-                Error message contains information about the invalid fields.""",
+                An empty error message means the order already exists with the current ID.
+                Otherwise, the error message contains information about the invalid fields.""",
```

Comment:
Need to discuss this behaviour with others, should we handle it the same way we handle products? Meaning if someone tries to create an order that already exists we return a 200 with existing order info. I think this would be more consistent.